### PR TITLE
cmd: added new output formats

### PR DESCRIPTION
**Pull request by ajnavarro**
Now you can change the output format to 2 new ones: json and csv.

- Added json support
- Added csv support
- Now the table output format is called 'pretty'. It is the default format.

### CODE REVIEW

**Comment by ajnavarro** [comment]
It Closes #82 and is related to #91

File: internal/format/csv.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	cw *csv.Writer
+}
+
+func NewCsvFormat(w io.Writer) *CsvFormat {
+	return &CsvFormat{
+		cw: csv.NewWriter(w),
+	}
+}
+
+func (cf *CsvFormat) Init(headers []string) error {
```
Comment (smola) [inline internal/format/csv.go]: These method names could look more standard:
Init -> WriteHeader
Line -> Write
End -> Close

File: internal/format/json.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+func (cf *JsonFormat) Init(headers []string) error {
+	cf.keys = headers
+
+	return nil
+}
+
+func (cf *JsonFormat) Line(line []interface{}) error {
+
```
Comment (smola) [inline internal/format/json.go]: extra line

File: internal/format/format_test.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+		Headers: []string{"a", "b", "c"},
+		Lines: [][]interface{}{
+			[]interface{}{
+				"a1", "b1", "c1",
+			},
+		},
+	},
+}
+
+func TestFormats(t *testing.T) {
```
Comment (smola) [inline internal/format/format_test.go]: use separate functions for each format so that we have separate test output for them

**Pull request merged**